source/documentation/administration_guide/topics/running_vnc_sasl_ansible_playbook_for_each_host.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+[id="running_vnc_sasl_ansible_playbook_for_each_host"]
+= Running the VNC SASL Ansible playbook for each host
+
+.Procedure
+
+. In the Administration Portal, put the hosts you created with FIPS enabled into maintenance mode:
+
+.. In the Administration Portal, click menu:Compute[Hosts].
+.. In the *Virtual Machines* column, verify that each host has zero virtual machines. Perform a live migration to remove all virtual machines from any hosts where necessary. For details, see link:{URL_downstream_virt_product_docs}virtual_machine_management_guide/index#sect-Migrating_Virtual_Machines_Between_Hosts[Migrating Virtual Machines Between Hosts]
+.. Select each host and click menu:Management[Maintenance] and btn:[OK].
+
+. Connect to the command line of the machine where the Manager is running.
++
+* If the Manager is running on a stand-alone machine, connect to that machine's command line. For example, enter:
++
+[source,terminal,subs="normal"]
+----
+# ssh root@rhvm
+----
+
+* If the Manager is running as a self-hosted engine, use menu:Compute[Virtual Machines] to select the self-hosted engine virtual machine, named `HostedEngine` by default, and click btn:[Console].
+
+. On the Manager machine, run the VNC SASL ansible playbook for each host.
++
+[source,terminal,subs="normal"]
+----
+# cd /usr/share/ovirt-engine/ansible-runner-service-project/project/
+# ansible-playbook --ask-pass --inventory=<hostname>, ovirt-vnc-sasl.yml
```

Comment:
@sandrobonazzola 
Should there be a comma in this command?

Reply:
I don't think so. @mnecas ?